src/main/scala/org/broadinstitute/dsde/rawls/model/GoogleBucketACL.scala

```diff
@@ -38,6 +38,7 @@ object GCSAccessLevel extends Enumeration {
   }
 }
 
+// TODO do we even need this?
```

Comment:
If we are going to change to implementation-of-security-agnostic approach (focusing on ACLs for workspace, not anything specific about buckets, google, groups, etc), I'd rename things like GCSAccessLevel, GoogleBucketACL, fromGoogleString, etc and just make them workspace ACLs